### PR TITLE
Calculator setup logic

### PR DESCRIPTION
**Pull request by touredri**
In this PR we have the following:

- [ ] Use props and useState
- [ ] Make operation possible

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -1,13 +1,35 @@
...
   </div>
 );
 
-const DisplayResult = () => <div className="calculator__display">0</div>;
-
+DisplayResult.propTypes = {
+  result: PropTypes.string.isRequired,
+  next: PropTypes.string.isRequired,
+  operation: PropTypes.string.isRequired,
+};
```
Comment (VICTOR-CHUKWU) [inline src/components/calculator.js]: - [ ] good job @touredri, you can still make this function a separate component and import it here too as you did with `CalculatorKey` :+1:

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @touredri , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed
- [ ] all event working correctly

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js (hunk 1 above)
Comment (touredri) [reply src/components/calculator.js]: Yeah @VICTOR-CHUKWU good idea

Comment (touredri) [reply src/components/calculator.js]: As it is not too long, cause I let there, interesting

**Pull request merged**